README.md

```diff
@@ -1,9 +1,18 @@
+
+### Firmware Packages & dentOS Tool Versions
 Packages for Mellanox and Marvell Firmware and specific versions of tools used to build dentOS
 
-mstpd
-ethtool 5.6.0
-iproute2 5.6.0
-lldpd 1.0.5
-mft 4.14.0-105
-realpath 8.26-3
-frr 6.0.3
+- mstpd
+- ethtool 5.6.0
+- iproute2 5.6.0
+- lldpd 1.0.5
+- mft 4.14.0-105
+- realpath 8.26-3
+- frr 6.0.3
+
+---
```

Comment:
The horizontal line is not necessary, in my opinion.

Reply:
I have removed it